src/EditorFeatures/CSharpTest/RemoveUnusedParametersAndValues/RemoveUnusedValueAssignmentTests.cs

```diff
@@ -1909,15 +1909,15 @@ void M(object p)
     {
         if (p is C [|x|])
         {
-            C x = null;
+            x = null;
```

Comment:
oh interesting.  so did we just have a full on bug here?

Reply:
Yes, the test had an error.